gui/src/ScenarioVisualizer.tsx

```diff
@@ -0,0 +1,652 @@
+/*
```

Comment:
"Visualizer" is actually too long, and impossible to pronounce.
Can you please rename that to "Viewer"?

-> Component: ScenarioViewer 
-> Control: scenario (in GuiCoreLib.py)